ERCS/erc-7007.md

```diff
@@ -56,34 +58,24 @@ pragma solidity ^0.8.18;
  */
```

Comment:
Please confirm that the above interface id is still correct.

Reply:
Thanks for pointing out. Fixed in https://github.com/ethereum/ERCs/pull/461/commits/23e5b00c66354299dfdc7a470aca7760967c5a7b